L1Trigger/L1TTwinMux/interface/DTLowQMatching.h

```diff
@@ -44,9 +45,9 @@ class DTLowQMatching  {
 
   void Matching(int track_seg);
 
-  L1MuTMChambPhContainer* m_phiDTDigis;
-  L1MuTMChambPhContainer m_phiRPCDigis;
-  L1MuDTChambPhContainer m_phiRPCDigis2;
+  L1MuDTChambPhContainer* m_phiDTDigis;
+  L1MuDTChambPhContainer m_phiRPCDigis;
```

Comment:
Why is one a pointer and the other one not?

Reply:
get container function in L1MuDTChamb do not return pointers. so we use it as it. is essentially comes from L1TTwinMuxAlgorithm.cc